osquery/distributed/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+ADD_OSQUERY_CORE_LIBRARY(osquery_distributed
+  distributed.cpp
+)
+
+ADD_OSQUERY_TEST(distributed_tests distributed_tests.cpp)
```

Comment:
The SDK merge changes this notation:
```
ADD_OSQUERY_LIBRARY(TRUE osquery_distributed
  distributed.cpp
)

ADD_OSQUERY_TEST(TRUE distributed_tests distributed_tests.cpp)
```

Where the first variable to the CMake macro indicates if it should be built as part of the SDK requirements. :)